src/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector.h

```diff
@@ -35,6 +35,10 @@
  *   server side.
  * - is_integrity_only: a flag indicating if the protector instance will be
  *   used for integrity-only or privacy-integrity mode.
+ * - enable_extra_copy: a flag indicating if the protector instance does one
+ *   extra memory copy during the protect operation for integrity_only mode.
```

Comment:
Could you add one more description that for unprotect operation, it is still zero-copy  to make it more clear?

Reply:
Good suggestion. Done